pkg/lib/config/messaging_whatsapp.go

```diff
@@ -32,8 +32,23 @@ var _ = Schema.Add("WhatsappTemplatesConfig", `
 }
 `)
 
+type WhatsappTemplateType string
+
+const (
+	WhatsappTemplateTypeAuthentication WhatsappTemplateType = "authentication"
+	WhatsappTemplateTypeUtil           WhatsappTemplateType = "util"
+)
+
+var _ = Schema.Add("WhatsappTemplateType", `
+{
+	"type": "string",
+	"enum": ["authentication", "util"]
```

Comment:
The way we support `util` is NOT language-agnostic. I suggest we only support authentication template type for now.